hack/run-functests.sh

```diff
@@ -25,9 +27,20 @@ echo "Focusing on $FOCUS"
 
 export SUITES_PATH=cnf-tests/bin
 
-if ! cnf-tests/test-run.sh -ginkgo.focus=$FOCUS -junit /tmp/artifacts/ -report /tmp/artifacts/; then
-  failed=true
-  failures+=( "Tier 2 tests for $FEATURES" )
+if [ $CNF_TESTS_IMAGE != "" ]; then
```

Comment:
missing "" around $CNF_TESTS_IMAGE
but actually a more resilient way to test if variable exist is:
```
if [ -n ${CNF_TESTS_IMAGE+x} ]; then
```

Reply:
What is the +x supposed to do?

Reply:
if CNF_TESTS_IMAGE exists, the "+" will replace it with "x" so no matter what the value is, it wont interfere with parsing the if statement.

this way also distinguish between unset variable and variable that equals ""

i guess a
-n "${CNF_TESTS_IMAGE}" 
is also fine in this case (as an empty value is meaningless
but actually gives a way of settinga default
ie - unset - run locally, 
set to "" mean use default 
anything else - use that..

Reply:
it works only with -z though
so either keep as is :-)
or do
! -z

Reply:
I'll keep as it is wrapping with ""